File: src/main/java/ar/edu/unq/spring/modelo/clasesDePersonajes/Picaro.java
package ar.edu.unq.spring.modelo.clasesDePersonajes;

import ar.edu.unq.spring.modelo.Personaje;
import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class Picaro extends Personaje {

    private int sigilo = 0;

    public Picaro() {}

    public Picaro(String nombre) {
        this.setNombre(nombre);
    }
}

File: src/main/kotlin/ar/edu/unq/spring/modelo/clasesDePersonajes/Picaro.kt
package ar.edu.unq.spring.modelo.clasesDePersonajes

import ar.edu.unq.spring.modelo.Personaje
import javax.persistence.Entity

@Entity
class Picaro() : Personaje() {
    var sigilo: Int = 0

    constructor(nombre: String) : this() {
        this.nombre = nombre
    }

}
